extensions/wikia/WallNotifications/WallNotificationsEveryone.class.php

```diff
@@ -167,28 +216,22 @@ public function getEntityProcessed( $userId, $entityKey ) {
 			__METHOD__
 		);
 
-		if ( $row->cnt == 0 ) {
-			wfProfileOut( __METHOD__ );
-			return false;
-		}
-
-		$this->setEntityProcessed( $userId, $entityKey );
 		wfProfileOut( __METHOD__ );
-		return true;
+		return ( $row->cnt == 0 ) ? false : true;
 	}
 
-	public function setQueueProcessed( $userId ) {
+	private function setQueueProcessed( $userId ) {
 		global $wgMemc;
 
 		wfProfileIn( __METHOD__ );
 
 		$cacheKey = $this->getQueueProcessedCacheKey( $userId );
-		$wgMemc->set( $cacheKey, true );
+		$wgMemc->set( $cacheKey, true ); // TODO: cache for infite time?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)